src/blogtoblog/BlogToBlogImporter.ts

```diff
@@ -280,13 +275,36 @@ export default class BlogToBlogImporter extends PageImporter {
     });
   }
 
-  rewriteLinks(main: Element): void {
+  rewriteLinks(main: Element, entries: any): void {
     main.querySelectorAll('a').forEach((a) => {
       const { href, innerHTML } = a;
       // TODO: use outer cdn URL
       if (href.startsWith('https://blog.adobe.com/')) {
-        a.href = href.replace('https://blog.adobe.com/', 'https://main--business-website--adobe.hlx3.page/');
-        a.innerHTML = innerHTML.replace('https://blog.adobe.com/', 'https://main--business-website--adobe.hlx3.page/');
+        // check linked blog posts
+        if (href.startsWith('https://blog.adobe.com/en/publish')) {
+          const title = href.split('/').pop().split('.').shift();
+          const match = entries.find((entry) => {
+            return entry.Target.includes(title);
+          });
+          if (match) {
+            // linked blog post is imported, update url
+            a.href = `https://main--business-website--adobe.hlx3.page${match.Target}`;
```

Comment:
Minor comment: before we had one place to replace `https://main--business-website--adobe.hlx3.page` by the final production URL. Now we start having plenty of those. 
We could create a global variable like:

```
const TARGET_HOST = 'https://main--business-website--adobe.hlx3.page'
```

and use the variable in the code. Before the final import, we only have place to change it ;)